docs/ports.md

```diff
@@ -4,43 +4,45 @@ title: "Services exposed and ports used"
 ---
 # Services exposed and ports used
 
-MicroK8s is meant to be used for local development thus certain security issues are not addressed at this point. Here we present the ports and sockets each service uses as well as the default authorisation and authentication configuration.
+MicroK8s is meant to be used for local development and so certain security issues are not addressed at this point. Here we present the ports and sockets each service uses as well as the default authorisation and authentication configuration.
 
-Services can be placed in two groups based on the network interface they bind to. Services binding to the localhost interface are only available from within the host and we take no action to limit access to them. Services binding to the default host interface are available from outside the host and thus we enforce some form of access restrictions.
+Services can be placed in two groups based on the network interface they bind to. Services binding to the `localhost`
+interface are only available from within the host and we take no action to limit access to them. Services binding to the
```

Comment:
I see you are adding new lines. Does this affect the layout? Some Juju charms look ugly when not in the right resolution because of the new lines. What is the recommendation here?

Reply:
99% of markdown doesn't care about newlines - only \n\n translates to an actual linefeed. wrapping the text makes it much easier to see the actual changes in a diff. I don't know what engine this site uses exactly but I believe it will be fine. I will fix it if not!